bin/network-monitor/src/config.rs

```diff
@@ -12,77 +74,88 @@ const DEFAULT_REMOTE_PROVER_URLS: &str = "http://localhost:50052";
 const DEFAULT_FAUCET_URL: &str = "http://localhost:8080";
 const DEFAULT_PORT: u16 = 3000;
 
-const RPC_URL_ENV_VAR: &str = "MIDEN_MONITOR_RPC_URL";
-const REMOTE_PROVER_URLS_ENV_VAR: &str = "MIDEN_MONITOR_REMOTE_PROVER_URLS";
-const FAUCET_URL_ENV_VAR: &str = "MIDEN_MONITOR_FAUCET_URL";
-const PORT_ENV_VAR: &str = "MIDEN_MONITOR_PORT";
-const ENABLE_OTEL_ENV_VAR: &str = "MIDEN_MONITOR_ENABLE_OTEL";
-
 /// Configuration for the monitor.
 ///
 /// This struct contains the configuration for the monitor.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Parser)]
+#[command(name = "miden-network-monitor")]
+#[command(about = "A network monitor for Miden node services")]
```

Comment:
nit: Current phrasing feels a bit awkward. Could remove "network" or rephrase it to say something like "A monitoring app for services in a Miden network".